File: cpp/26_移除元素_easy.cpp
class Solution {
public:
    int removeElement(vector<int>& nums, int val) {
        int n = nums.size();
        if (!n) return 0;
        int index = 0;
        for (int i=0; i<n; i++)
            if (nums[i] != val)
                nums[index++] = nums[i];
        return index;
    }
};

File: python/26_移除元素_easy.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        n, index = len(nums), 0
        if not n:
            return 0
        for i in range(n):
            if nums[i] != val:
                nums[index] = nums[i]
                index += 1
        return index